translation/lint.rst

```diff
@@ -0,0 +1,31 @@
+.. index::
+    single: Translation; Lint
+    single: Translation; Translation File Errors
+
+How to Find Errors in Translation Files
```

Comment:
but in the 3.2 and master versions of this article (which is already merged) we explain yaml and xliff. Besides, in the future we could explain more formats, so I think we should maintain the generic title.